.github/workflows/build-balena-disk-image.yaml

```diff
@@ -0,0 +1,60 @@
+name: Build Balena Disk Images
+on: workflow_dispatch
```

Comment:
- Does this mean that this job can be executed on demand?
- Will this job be used by other jobs?